packages/@expressive-code/plugin-shiki/src/index.ts

```diff
@@ -271,3 +370,13 @@ function getRemovedRanges(original: string, edited: string): [start: number, end
 
 	return ranges
 }
+
+async function createRegexEngine(engine: PluginShikiOptions['engine']) {
+	// The [...engine...][0] syntax makes it easier to find this code in the built package,
+	// allowing astro-expressive-code to remove unused engines from the SSR bundle
+	// TODO: This could be adjusted to use direct imports if/when astro-expressive-code is updated
```

Comment:
If `astro-expressive-code` is updated to use `Core`, these can become static imports vs. dynamic